assets/i18n/qqq.json

```diff
@@ -344,6 +344,7 @@
 	"I18N_PLAYER_HINT_REQUEST_STRING_3": "Text displayed when the user requests a hint while playing an exploration. - This text is displayed in the conversation interface when a learner clicks on the hint button.",
 	"I18N_PLAYER_INFO_TOOLTIP": "Text displayed when the user is playing an exploration. - Tooltip for the button that opens the a dialog with more information about the exploration.\n{{Identical|Info}}",
 	"I18N_PLAYER_LAST_UPDATED_TOOLTIP": "Text displayed when the user is playing an exploration. - When the user clicks the information button, a dialog opens with further information. This text is shown in that dialog as tooltip of the exploration last update.\n{{Identical|Last updated}}",
+	"I18N_PLAYER_LEARN_AGAIN_BUTTON": "Text displayed when the user is playing an exploration. - Text shown in a button. Only shown when current answer is wrong, next card is already visited and correctness_feedback is enabled. When the user clicks the button, the next card on the exploration is loaded.\n{{Identical|Continue}}",
```

Comment:
I don't think you want the "Identical" tag at the end

Reply:
Ok, removed